### ISSUE TITLE
Add'l details screen reader output Are you working with a CISA regional representative on your domain request?

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2921]
![Screenshot 2024-10-08 at 3 30 47 PM](https://github.com/user-attachments/assets/aba8c308-e37e-42fe-9e6b-af6a04d4747e)
### The screen reader output is:
Has cisa representative * No checked, required, 2 of 2

### It should be:
**Legend**: Are you working with a CISA regional representative on your domain request?
**aria-describedby**: .gov is managed by the Cybersecurity and Infrastructure Security Agency. CISA has [[10 regions](https://www.cisa.gov/about/regions)](https://www.cisa.gov/about/regions)
 that some organizations choose to work with. Regional representatives 
use titles like protective security advisors, cyber security advisors, 
or election security advisors.
**Label**: Yes / No
**required**: not checked, required, 2 of 2

**What it will look like all together**: Are you working with a CISA regional representative on your domain request? .gov is managed by the Cybersecurity and Infrastructure Security Agency. CISA has [[10 regions](https://www.cisa.gov/about/regions)](https://www.cisa.gov/about/regions) that some organizations choose to work with. Regional representatives use titles like protective security advisors, cyber security advisors, or election security advisors. Yes not checked, required, 2 of 2

### Prioritization Comments
**WCAG Conformance Level:** 2.1 AA
**WCAG Standard that applies:** 
1.3.1 - Info and Relationships (Critical)
3.3.2 - Labels or Instructions (Critical)
4.1.2 - Name, Role, Value (Critical)
**Critical Failure if Not Met?:**  Yes
**Prioritization:** Blocker
**Reason:** The incorrect labeling and lack of a clear description for the options "Yes" and "No" make it challenging for screen reader users to understand the form's structure and selection status. Correct labels and descriptions are essential to provide users with an accessible, consistent, and understandable experience.
